tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/RegisterDependencies.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BoDi;
+using NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Utils;
+using Microsoft.Extensions.Configuration;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests
+{
+    [Binding]
+    public sealed class RegisterDependencies
+    {
+        [BeforeScenario]
+        public void Register(IObjectContainer objectContainer)
+        {
+            var config = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .AddEnvironmentVariables()
+                .Build();
+
+            Database.ResetUser(config.GetConnectionString("CatalogueUsersAdmin"));
```

Comment:
suggest this is consistent with clearing the data. Either reset the user & clear the db both before or after the run.
Thinking about it, this will get run before each test... It may be better to try registering the instance [BeforeTestRun], so it's done once per run, as opposed to per test.